includes/definitions/wisi-tangram.yaml

```diff
@@ -0,0 +1,15 @@
+os: wisi-tangram
+text: 'WISI Tangram'
+type: network
+icon: wisi
+mib_dir:
+    - wisi
+over:
+    - { graph: '', text: '' }
```

Comment:
Did you forgot to remove or replace this?

Reply:
@TheGreatDoc I dont know what to put in there. I guess it could be fan sensors? This thing is horrible for snmp support..basically none of the interfaces show up besides the managment, no CPU, etc.. I got all I could out of it

Reply:
just make it an empty array `over: []` ?

Reply:
Cant he just delete the over section?

Reply:
@TheGreatDoc I tried deleting it and it threw some errors on discovery . I noticed another OS with that same style so i just copied that.. Making it a empty array works fine.. I'll change it.